openpype/tools/loader/model.py

```diff
@@ -1173,9 +1174,9 @@ def _on_doc_fetched(self):
                     repre_groups_items[doc["name"]] = 0
                     group = group_item
 
-            progress = lib.get_progress_for_repre(
-                doc, self.active_site, self.remote_site
-            )
+            progress = self.sync_server.get_progress_for_repre(
+                self.project_name, doc["_id"],
```

Comment:
Rather use `self.dbcon.active_project()` here and don't add new variable `project_name`.